sources/assembly-data/ATTR_assembly.types.yaml

```diff
@@ -116,35 +146,46 @@ attributes:
       count: 10
       scale: log10
   protein_count:
+    description: Protein count
+    long_description: Number of proteins in an assembly derived from annotation
```

Comment:
think there's no need to include "derived from annotation". An alternative would be to say "in an annotated assembly".